packages/mwp-app-route-plugin/src/route.js

```diff
@@ -40,20 +271,20 @@ export const onPreResponse = {
  *   used by Hapi
  */
 export default (languageRenderers: { [string]: LanguageRenderer$ }) => ({
-	method: 'GET',
-	path: '/{wild*}',
+	method: "GET",
+	path: "/{wild*}",
 	config: {
 		ext: {
-			onPreResponse,
+			onPreResponse
 		},
 		plugins: {
-			'electrode-csrf-jwt': {
-				enabled: true, // need to generate tokens on page request
-			},
+			"electrode-csrf-jwt": {
+				enabled: true // need to generate tokens on page request
+			}
 		},
 		state: {
-			failAction: 'ignore', // ignore cookie validation, just accept
-		},
+			failAction: "ignore" // ignore cookie validation, just accept
+		}
 	},
-	handler: getHandler(languageRenderers),
+	handler: getHandler(languageRenderers)
```

Comment:
throw an error in the handler function in order to get your custom 500 response

```js
	handler: (request, reply) => {
		reply(new Error());
	}, //getHandler(languageRenderers)
```

<img width="741" alt="screen shot 2017-12-19 at 1 12 54 pm" src="https://user-images.githubusercontent.com/1885153/34134242-5b5dbcda-e4be-11e7-889a-359d8d47f4e3.png">